pkg/services/live/live.go

```diff
@@ -384,11 +387,31 @@ func (g *GrafanaLive) getStreamPlugin(pluginID string) (backend.StreamHandler, e
 }
 
 func (g *GrafanaLive) Run(ctx context.Context) error {
+	eGroup, eCtx := errgroup.WithContext(ctx)
+
+	eGroup.Go(func() error {
+		updateStatsTicker := time.NewTicker(time.Minute * 30)
+		defer updateStatsTicker.Stop()
+
+		for {
+			select {
+			case <-updateStatsTicker.C:
+				g.sampleLiveStats()
+			case <-updateStatsTicker.C:
```

Comment:
Duplicate read from the same channel.

Reply:
Good catch! Copy-paste driven development is discouraged! 😛

Reply:
Fixed, thanks!